.env

```diff
@@ -13,8 +13,8 @@ MON_KEYSTONE_VERSION=1.1.1
 MON_KAFKA_VERSION=0.9.0.1-2.11-1.1.0
 MON_KAFKA_INIT_VERSION=0.0.1
 MON_ALARMS_VERSION=1.1.0
+MON_GRAFANA_INIT_VERSION=1.2.1
```

Comment:
Why are you moving this line?

Reply:
Sorry for late update.
This is my mistake when creating patch.
I re-move this variable to original line.